orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/TokenService.java

```diff
@@ -249,7 +249,9 @@ private SignedJWT generateIDToken(
             idTokenClaims.setClaim("vot", vot);
         }
         idTokenClaims.setClaim("vtm", trustMarkUri.toString());
-        idTokenClaims.setClaim("auth_time", authTime);
+        if (configService.isReturnAuthTimeInIdTokenEnabled()) {
+            idTokenClaims.setClaim("auth_time", authTime);
+        }
```

Comment:
Do we not want to put this inside the ` if (!isDocAppJourney)` check above, as we're not returning this claim for docapp users are we?

Reply:
Shouldn't matter as if it's a Doc App journey then auth_time is `null` but I'll do it anyway

Reply:
Done